### DISCUSSION TITLE
Welcome to UpSnap Discussions!

### DISCUSSION THREAD

**Discussion by seriousm4x** [seriousm4x/UpSnap#39] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cassieyu647-ai** [comment]
The search function in version 5.1.7 is not working. Could someone kindly help? Thank you very much.

**Comment by 5637487-trsdt** [comment]
Currently, I have to add devices to groups one at a time… my mouse is starting to give me side-eye. Any chance we could batch this magic? ✨  【 bulk actions for device management】
Huge thanks for all your hard work — seriously, you're awesome! 🙌💖

**Comment by Lintionlai** [comment]
Some computers have firewalls enabled, so they cannot be pinged, but other ports are accessible, which means the hosts are powered on. Why does the power button not turn green? Does the system require successful ping as the sole criterion? It should turn the button green if either ping succeeds or the specified ports are reachable.
<img width="766" height="305" alt="P1" src="https://github.com/user-attachments/assets/766deb89-8f74-49f3-b58e-33fe54d54d8d" />

**Reply by seriousm4x** [reply]
Only the device ping is responsible for the up/down status. It would create a lot of state management issues if every device port could set the device status along with the actual device ping.
